runtime/compiler/control/CompilationThread.cpp

```diff
@@ -3116,8 +3116,13 @@ void TR::CompilationInfo::stopCompilationThreads()
 
       fprintf(stderr, "Number of compilations per level:\n");
       for (int i = 0; i < (int)numHotnessLevels; i++)
+         {	      
          if (_statsOptLevels[i] > 0)
-            fprintf(stderr, "Level=%d\tnumComp=%d\n", i, _statsOptLevels[i]);
+            fprintf(stderr, "Level=%d\tnumComp=%d\tnumRemoteComp=%d\n", i, _statsOptLevels[i], _statsRemoteOptLevels[i]);
```

Comment:
This will not work on a build that does not have `J9VM_OPT_JITSERVER` defined because `_statsRemoteOptLevels` is going to be undefined.